django-s3-file-field-client/src/client.ts

```diff
@@ -71,7 +65,21 @@ export default class S3FFClient {
     let index = 0;
```

Comment:
Could you rename `index` to be something more descriptive now, like `file_offset` (or similar)?